README.md

```diff
@@ -1 +1,64 @@
-# Hello-Microverse-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Hello Microverse project.
+
+
+## Built With
+
+- Html and Css
+- No Frameworks
+- Linters 
+
+
+
+
+## Getting Started
+
+
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
**OPTIONAL** - To make your project even more professional, please include steps on how to clone, navigate into and get your project running locally 😄